### DISCUSSION TITLE
How to address Go to File sort order

### DISCUSSION THREAD

**Discussion by bcardarella** [zed-industries/zed#11936] [Feature Requests]
In an Elixir project I type `mix` for `CMD+P` and would expect `mix.exs` or `mix.lock` to match first but other files that appear to have all of the letters just not in the same word sort higher.

<img width="410" alt="Screenshot 2024-05-16 at 5 02 22 PM" src="https://github.com/zed-industries/zed/assets/18524/11aff26d-abb5-4174-9290-4d9d6c488d51">

**Comment by SomeoneToIgnore** [comment]
There's a small separator trying to indicate that what's above are history items

<img width="130" alt="image" src="https://github.com/zed-industries/zed/assets/2690773/6a5e7f8c-385f-47a3-9e84-015f7b54ff03">

and we show all history items with any matches above others currently, without alternatives, as VSCode and some other editors do.

**Comment by bcardarella** [comment]
Can that be turned off? If I'm opening a new file odds are I don't want from history.

**Comment by SomeoneToIgnore** [comment]
As said,

> without alternatives

so no.
Feel free to submit an issue or the PR.

**Comment by SomeoneToIgnore** [comment]
Actually, here's the newly created one: https://github.com/zed-industries/zed/issues/11970
